designs/2019-eslintrc-improvements/README.md

```diff
@@ -0,0 +1,334 @@
+- Start Date: 2019-02-25
+- RFC PR: (leave this empty, to be filled in later)
+- Authors: (the names of everyone contributing to this RFC)
+
+# Config File Improvements
+
+## Summary
+
+This proposal improves the internal logic of configuration files to maintain our codebase easier and add enhancements easier. This is a change of architecture and large refactoring.
```

Comment:
It seems like this RFC has a lot of refactoring, and also some user-facing effects. Could you summarize what the main user-facing effects are?

From reading through quickly, I saw the following changes, but I might have missed some:

* Updates the `files` option to determine the set of files that get linted
* Allows `extends` to be used in `overrides`
* Loads plugins relative to config files, rather than the CWD? (This might have been caused by confusion from https://github.com/eslint/rfcs/pull/14#issuecomment-466882086.)
* Fixes a bug in `overrides`/`extends` resolution order
* Loads parsers lazily

Reply:
I will update the summary. I'm sorry.

Reply:
I updated it.